etc/cas.properties

```diff
@@ -37,6 +37,7 @@ google.analytics.autoLink=
 cas.osf.login.url=/login?
 cas.logout.url=/logout
 cas.institution.login.url=/login?campaign=institution
+cas.institution.unsupported.url=/login?campaign=unsupportedinstitution
```

Comment:
As we discussed during campfire, if CAS is logged out but OSF is logged in, OSF may not handle the form post correctly. This shouldn't happen at all unless user removed CAS cookie manually. The all-in-one solution is to build the URL (not here but in the JSP) with OSF `logout` as the first endpoint to hit. The `next` URL will be the CAS url (with service param). Doing URL encoding in JSP is ugly but we can send the encoded version (with service) via `loginContext`.

Extra note: with "Stay signed in" checked, CAS has a persistent cookie; with it unchecked, CAS has a session cookie which will be destroyed at the same time OSF's session cookie is destroyed. This is why the case above shouldn't happen in normal work flow. However, both CAS and OSF should be able to handle this case in robust way.